visualization/dashboard/src/MLIDashboard/NewExplanationDashboard.tsx

```diff
@@ -297,9 +326,21 @@ export class NewExplanationDashboard extends React.PureComponent<IExplanationDas
                         cohorts={this.state.cohorts}
                         jointDataset={this.state.jointDataset}
                         metadata={this.state.modelMetadata}
-                        onChange={this.onCohortChange}
-                        onDelete={this.deleteCohort}
+                        editCohort={this.openCohort}
+                        cloneAndEdit={this.cloneAndOpenCohort}
                     />
+                    {cohortForEdit !== undefined && (
+                        <CohortEditor
+                            jointDataset={this.state.jointDataset}
+                            filterList={cohortForEdit.filterList}
+                            cohortName={cohortForEdit.cohortName}
+                            onSave={this.onCohortChange}
+                            onCancel={this.closeCohortEditor}
+                            onDelete={this.deleteCohort}
+                            isNewCohort={this.state.editingCohortIndex === this.state.cohorts.length}
+                            deleteIsDisabled={this.state.cohorts.length === 1}
```

Comment:
The idea here is that the starting full set of data is the original cohort?

Reply:
the starting default cohort is the full dataset, the idea is to prompt the user to look at subsets if the full dataset is too big for some charts